.yarn/patches/async-done-npm-1.3.2-1f0a4a8997.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/index.js b/index.js
```

Comment:
[Endo-SES](https://github.com/endojs/endo/) new version introduces "domain taming" which prevents systems from using the node builtin "domain" package due to security reasons.
Here we remove the usage "async-done" package is making of "domain" package so MM build system continues to work properly.